docs/reference.rst

```diff
@@ -6,7 +6,7 @@ RemoteFrameBuffer class
 -----------------------
 
 .. autoclass:: jupyter_rfb.RemoteFrameBuffer
-    :members: request_draw, get_frame, handle_event, reset_stats, get_stats
+    :members:
```

Comment:
I'm not sure I brought this up yet, but what about `sphinxcontrib.apidoc` to auto-generate the API files? Or is it small enough that that's overkill.

Reply:
There's just two autodoc entries, so I'd consider it overkill indeed :)